docs/develop/enable-nested-app-authentication-in-your-add-in.md

```diff
@@ -237,35 +237,35 @@ The following table shows which APIs are supported when NAA is enabled in the MS
 
 | Method                        | Supported by NAA      |
 | ----------------------------- | --------------------- |
-| _acquireTokenByCode_          | NO (throws exception) |
-| _acquireTokenPopup_           | YES                   |
-| _acquireTokenRedirect_        | NO (throws exception) |
-| _acquireTokenSilent_          | YES                   |
-| _addEventCallback_            | YES                   |
-| _addPerformanceCallback_      | NO (throws exception) |
-| _disableAccountStorageEvents_ | NO (throws exception) |
-| _enableAccountStorageEvents_  | NO (throws exception) |
-| _getAccountByHomeId_          | YES                   |
-| _getAccountByLocalId_         | YES                   |
-| _getAccountByUsername_        | YES                   |
-| _getActiveAccount_            | YES                   |
-| _getAllAccounts_              | YES                   |
-| _getConfiguration_            | YES                   |
-| _getLogger_                   | YES                   |
-| _getTokenCache_               | NO (throws exception) |
-| _handleRedirectPromise_       | NO                    |
-| _initialize_                  | YES                   |
-| _initializeWrapperLibrary_    | YES                   |
-| _loginPopup_                  | YES                   |
-| _loginRedirect_               | NO (throws exception) |
-| _logout_                      | NO (throws exception) |
-| _logoutPopup_                 | NO (throws exception) |
-| _logoutRedirect_              | NO (throws exception) |
-| _removeEventCallback_         | YES                   |
-| _removePerformanceCallback_   | NO (throws exception) |
-| _setActiveAccount_            | NO                    |
-| _setLogger_                   | YES                   |
-| _ssoSilent_                   | YES                   |
+| *acquireTokenByCode*          | NO (throws exception) |
```

Comment:
Instead of italic, should these method names be in code ticks?

Reply:
fixed